fastlane/Fastfile

```diff
@@ -240,6 +269,26 @@ platform :android do
     )
   end
 
+  lane :build_qa do
+    prepare_release
+
+    build_android
+
+    abbreviated_commit_hash = last_git_commit[:abbreviated_commit_hash]
+
+    s3 = Aws::S3::Resource.new(region: ENV['AWS_REGION'])
+
+    apk_obj = s3.bucket(ENV['AWS_BUCKET_NAME']).object("android/#{abbreviated_commit_hash}.apk")
+    apk_obj.upload_file("#{lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]}")
+
+    android_install_path = "https://s3.#{ENV['AWS_REGION']}.amazonaws.com/#{ENV['AWS_BUCKET_NAME']}/android/#{abbreviated_commit_hash}.apk"
+
+    if ENV['MATTERMOST_WEBHOOK_URL']
+      msg = "QA build [#{abbreviated_commit_hash}](https://github.com/mattermost/mattermost-mobile/commit/#{abbreviated_commit_hash}) — Android: #{android_install_path}"
+      mattermost(message: msg, username: 'Fastlane', mattermost_url: ENV['MATTERMOST_WEBHOOK_URL'])
```

Comment:
I can also make `MATTERMOST_URL` an alias for `MATTERMOST_WEBHOOK_URL` as an environment variable if anyone prefers that. I have no preference.

Reply:
yeah I'm 0/5 on that I know `MATTERMOST_WEBHOOK_URL` is more verbose but then is not ambiguous where a `MATTERMOST_URL` can be confused with the server url